internal/kubernetes/operator/config_exporter_test.go

```diff
@@ -416,3 +416,230 @@ func TestExportAtlasStreamProcessing(t *testing.T) {
 		)
 	})
 }
+func setupMockController(t *testing.T) (*gomock.Controller, *mocks.MockOperatorGenericStore, *ConfigExporter) {
```

Comment:
Nit pick:
I tend to put helpers at the end, as they are less interesting up front. In a test file you want to see the tests cases themselves ASAP, usually.